- Don't install libnetpbm.a and libnetpbm.so into a sep link dir
- Try to fix parallel building
- Make the libopt tool less 'helpful'

--- lib/Makefile
+++ lib/Makefile
@@ -249,7 +249,7 @@
 .PHONY: install.staticlib
 install.staticlib: $(PKGDIR)/link
 	$(INSTALL) -c -m $(INSTALL_PERM_LIBS) libnetpbm.$(STATICLIBSUFFIX) \
-	  $(PKGDIR)/link
+	  $(PKGDIR)/lib
 
 # Install a shared library stub -- the ".so" file used at link time to
 # prepare a program for dynamically linking a library at run time 
@@ -257,9 +257,9 @@
 install.sharedlibstub: $(PKGDIR)/link
 ifeq ($(NETPBMLIBTYPE),unixshared)
 # install the link-time (.so) links to the runtime libraries
-	cd $(PKGDIR)/link ; \
+	cd $(PKGDIR)/lib ; \
           rm -f libnetpbm.$(NETPBMLIBSUFFIX); \
-          $(SYMLINK) ../lib/libnetpbm.$(NETPBMLIBSUFFIX).$(MAJ) \
+          $(SYMLINK) libnetpbm.$(NETPBMLIBSUFFIX).$(MAJ) \
             libnetpbm.$(NETPBMLIBSUFFIX)
 endif
 ifeq ($(NETPBMLIBTYPE),dll)
--- GNUmakefile
+++ GNUmakefile
@@ -87,6 +87,9 @@
 
 .PHONY: nonmerge
 nonmerge: $(PRODUCT_SUBDIRS:%=%/all)
+
+$(PRODUCT_SUBDIRS:%=%/all): buildtools/all
+converter/all analyzer/all editor/all generator/all other/all: lib/all
 
 OMIT_CONFIG_RULE = 1
 include $(SRCDIR)/Makefile.common
--- buildtools/libopt.c
+++ buildtools/libopt.c
@@ -502,7 +502,7 @@
 
     strcpy(outputLine, "");  /* initial value */
     runtime = FALSE;  /* initial value */
-    quiet = FALSE;   /* initial value */
+    quiet = TRUE;   /* initial value */
     error = FALSE;  /* no error yet */
     for (arg = 1; arg < argc && !error; arg++) {
         if (strcmp(argv[arg], "-runtime") == 0)
